package.json

```diff
@@ -36,10 +36,13 @@
   "dependencies": {
     "@hapi/joi": "^16.1.4",
     "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
     "env2": "^2.2.2",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
-    "pg": "^7.12.1"
+    "pg": "^7.12.1",
+    "react-router-dom": "^5.1.1",
```

Comment:
Why is `react-router-dom` and `react-stars` installed in the backend?!?!?!?!!?!??!?!?!
It's obviously a mistake and not intended, but you must fix it before pushing it!!